packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -2095,6 +2095,21 @@ class _FifoCache<K, V> {
 /// the list. For chips, it only affects the vertical size, not the horizontal
 /// size.
 ///
+/// Here are some examples of widgets that respond to density changes:
+///  - [Checkbox]
```

Comment:
you'll need a blank line here I believe or it'll just become a big paragraph

Reply:
<*shakes fist*> darn you markdown!